src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -723,10 +723,10 @@ public void verifyTablePattern(int tableNum, int column, String patternString){
      * @return The page (for chaining method calls).
      */
     public AppPage verifyHtml(String filePath) throws IOException {
-        return verifyHtml(null, filePath, false);
+        return verifyHtml(null, filePath);
     }
 
-    private AppPage verifyHtml(By by, String filePath, boolean isWithRetry) throws IOException {
+    private AppPage verifyHtml(By by, String filePath) throws IOException {
```

Comment:
I've been wondering as well. There's no chain method call after `verifyHtml*` (and it doesn't make sense to make one anyway).